src/internal/connector/onedrive/drive_test.go

```diff
@@ -49,22 +47,22 @@ func (suite *OneDriveSuite) TestCreateGetDeleteFolder() {
 
 	driveID := *drives[0].GetId()
 
-	folderID, err := createRestoreFolders(ctx, gs, driveID, folderElements)
-	require.NoError(t, err)
-
-	folderIDs = append(folderIDs, folderID)
+	names := []string{}
+	ids := []string{}
 
-	defer func() {
-		assert.NoError(t, DeleteItem(ctx, gs, driveID, folderIDs[0]))
-	}()
+	for i := 0; i < 2; i++ {
+		name := "Corso_Folder_Test_" + common.FormatNow(common.SimpleTimeTesting)
+		folderElements := []string{name}
+		id, err := createRestoreFolders(ctx, gs, driveID, folderElements)
+		require.NoError(t, err)
 
-	folderName2 := "Corso_Folder_Test_" + common.FormatNow(common.SimpleTimeTesting)
-	folderElements = append(folderElements, folderName2)
```

Comment:
Reducing folderElements to a single folder is the change that 'fixes' the tests.  We could, if wanted, bring this back by updating line 98 to check for a pre-determined length (count of the expected restored folders).  Doesn't quite explain why this only bugs out for particular users (AdeleV).

Reply:
@vkamra Unless you want us to keep it, I'm going to close this ticket.  Now that we're unable to repro the error, I'm leaning towards the assumption that it was a temporarily bad data state for the given test user.

Reply:
Let's close this and I'm going to look to see if I can repro locally to confirm the Delta behavior we discussed